src/Controllers/api/v2.php

```diff
@@ -40,13 +40,18 @@
 
 try {
     $response = MyRadio_Swagger2::handleRequest($op, $class, $method, $id, $arg0);
+    if ($response['status']) {
+        header('HTTP/1.1 ' . $response['status']);
+    }
+    header('Content-Type: application/json');
```

Comment:
Does HTTP/1.1 header have to come first? otherwise it looks like the Content-Type can be outside the try block